### ISSUE TITLE
Creep is not afraid of vampire players.

### ISSUE THREAD

**Issue by GTX1080TiBOOM** [TeamLapen/Vampirism#1445]
### Minecraft Version

1.20.x

### Forge Version

47.3.11

### Vampirism Version

1.10.0

### Incompatible mod

Grimoire of Gaia 4

### Singleplayer/Multiplayer

Singleplayer

### What happened?

Unlike Creeper, Creep is only afraid of cats.

### Reproduce steps

![2024-10-29_12 42 55](https://github.com/user-attachments/assets/f31f0dd9-be84-4f6a-9a88-015312b4cad2)

**Comment by Cheaterpaul** [comment]
Yes, a creep is not a Creeper. Thus they are not afraid of vampires.
